queue/driver_mongo.go

```diff
@@ -419,8 +474,6 @@ func (d *mongoDriver) Get(ctx context.Context, name string) (amboy.Job, error) {
 		return nil, errors.Wrapf(err, "GET problem decoding '%s'", name)
 	}
 
-	d.processNameForUsers(j)
```

Comment:
Should this call `d.removeMetadata()`?

Reply:
?

Reply:
I mentioned this in the original PR comment but this code is not necessary. All it does is modify the job interchange name, which is not necessary if all you want is to return the Job object.